Makefile

```diff
@@ -25,6 +27,7 @@ install-data:
 	install -dDm0755 $(DESTDIR)/$(VARDIR)/lib/speakersafetyd/blackbox
```

Comment:
blackbox also needs to be owned by speakersafetyd user

Reply:
Makefile now installs this as owned by speakersafetyd and tmpfiles.d will create it owned by speakersafetyd